Jenkinsfile_CNP

```diff
@@ -48,11 +48,11 @@ withPipeline(type, product, component) {
 
     after('checkout') {sh 'yarn cache clean'}
 
-// before('buildinfra:prod') {
-  // currentBuild.result = 'SUCCESS'
-  // currentBuild.description = "SUCCESS"
-  // error 'XUI  projects are not authorised to be in production environments yet.'
-// }
+ before('buildinfra:prod') {
+ currentBuild.result = 'SUCCESS'
+ currentBuild.description = "SUCCESS"
+ error 'XUI  projects are not authorised to be in production environments yet.'
```

Comment:
Change message to say "ExUI Approve Org production deployment is currently disabled"